src/support.ts

```diff
@@ -26,8 +26,12 @@ export interface Support {
  */
 export function getSupport(feature: string, browser = currentBrowser): Support {
   const support: Support = { level: "unknown", needPrefix: false, notes: [] };
-  const stats = db.data[feature].stats[browser.id];
-  if (!stats) { return support; }
+  let stats;
+  if (db.data[feature].stats[browser.id]) {
+    stats = db.data[feature].stats[browser.id]
+  } else {
+    return support;
+  }
```

Comment:
What are you trying to fix? It looks to me as it does the same thing.

Reply:
my bad, wanted to check this way:
```
var feature = db.data[feature];
var browserId; 
if (feature && feature.stats) {
    browserId = feature.stats[browser.id];
} else {
    return support;
}
```

Reply:
It needed to make work with props which not exist in db or not have `stats` prop in `feature` object. This change will help to remove this unnecessary `filter` - https://github.com/cssinjs/css-vendor/blob/master/test/fixtures/property-prefix.js#L68